packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -1498,7 +1498,6 @@ describe('ReactDOMServerPartialHydration', () => {
     expect(container.textContent).toBe('ALoading B');
   });
 
-  // @gate experimental || www
```

Comment:
Removed this gating because unstable_scheduleHydration is no longer gated through `index.stable.js` because its exposed through ReactDOMHydrationRoot instead of the ReactDOM package

Reply:
(previously the test would fail because `unstable_scheduleHydration` isn't exposed by `index.stable.js`

Reply:
you probably want to gate on enableSuspenseList now?

Reply:
Yep just pulled and merged